app/code/Magento/Catalog/Controller/Adminhtml/Product/Action/Attribute/Save.php

```diff
@@ -52,72 +39,89 @@ class Save extends \Magento\Catalog\Controller\Adminhtml\Product\Action\Attribut
-     * @param \Magento\CatalogInventory\Api\StockConfigurationInterface $stockConfiguration
      * @param \Magento\CatalogInventory\Api\Data\StockItemInterfaceFactory $stockItemFactory
      * @param \Magento\Framework\Api\DataObjectHelper $dataObjectHelper
+     * @param \Magento\Framework\Stdlib\DateTime\TimezoneInterface|null $localeDate
+     * @param \Magento\Eav\Model\Config|null $eavConfig
+     * @param \Magento\CatalogInventory\Api\StockRegistryInterface|null $stockRegistry
+     * @param \Magento\CatalogInventory\Api\StockItemRepositoryInterface|null $stockItemRepository
+     * @param \Magento\CatalogInventory\Api\StockConfigurationInterface|null $stockConfiguration
      */
     public function __construct(
         Action\Context $context,
-        \Magento\Framework\Stdlib\DateTime\TimezoneInterface $localeDate,
-        \Magento\Eav\Model\Config $eavConfig,
         \Magento\Catalog\Helper\Product\Edit\Action\Attribute $attributeHelper,
         \Magento\Catalog\Model\Indexer\Product\Flat\Processor $productFlatIndexerProcessor,
         \Magento\Catalog\Model\Indexer\Product\Price\Processor $productPriceIndexerProcessor,
         \Magento\CatalogInventory\Model\Indexer\Stock\Processor $stockIndexerProcessor,
         \Magento\Catalog\Helper\Product $catalogProduct,
-        \Magento\CatalogInventory\Api\StockRegistryInterface $stockRegistry,
-        \Magento\CatalogInventory\Api\StockItemRepositoryInterface $stockItemRepository,
-        \Magento\CatalogInventory\Api\StockConfigurationInterface $stockConfiguration,
         \Magento\CatalogInventory\Api\Data\StockItemInterfaceFactory $stockItemFactory,
-        \Magento\Framework\Api\DataObjectHelper $dataObjectHelper
+        \Magento\Framework\Api\DataObjectHelper $dataObjectHelper,
+        \Magento\Framework\Stdlib\DateTime\TimezoneInterface $localeDate = null,
+        \Magento\Eav\Model\Config $eavConfig = null,
+        \Magento\CatalogInventory\Api\StockRegistryInterface $stockRegistry = null,
+        \Magento\CatalogInventory\Api\StockItemRepositoryInterface $stockItemRepository = null,
+        \Magento\CatalogInventory\Api\StockConfigurationInterface $stockConfiguration = null
     ) {
-        $this->localeDate = $localeDate;
-        $this->eavConfig = $eavConfig;
         $this->_productFlatIndexerProcessor = $productFlatIndexerProcessor;
         $this->_productPriceIndexerProcessor = $productPriceIndexerProcessor;
         $this->_stockIndexerProcessor = $stockIndexerProcessor;
         $this->_catalogProduct = $catalogProduct;
-        $this->stockRegistry = $stockRegistry;
-        $this->stockItemRepository = $stockItemRepository;
-        $this->stockConfiguration = $stockConfiguration;
         $this->stockItemFactory = $stockItemFactory;
         parent::__construct($context, $attributeHelper);
         $this->dataObjectHelper = $dataObjectHelper;
+        $this->localeDate = $localeDate ?: \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
I'm using netbeans, so I do not think I can do that.
I did look at what PhpStorm did, so I tried to mimic it.